Trivy 0.24.0 Segfaults on Terraform config scan

## Description

Trivy 0.24.0 and 0.24.1 throws a segfault when I run `trivy config`. Trivy 0.23.0 works without segfaulting. 

I am running Trivy inside the aquasec/trivy container.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xf33beb]

goroutine 1 [running]:
github.com/aquasecurity/defsec/rules.(*Result).Flatten(_)
        /home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.12.1/rules/flat.go:61 +0x32b
github.com/aquasecurity/fanal/config/scanner.Scanner.scanTerraform({{0x7ffe6831cf3b, 0x2b}, {0xc0004cd320, 0x2, 0x2}, 0xc0008746c0, 0xc00083c0e0, 0xc0009de230}, {0xc00032c900, 0xe, ...})
        /home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220225095822-ef150f781751/config/scanner/scanner.go:227 +0x425
github.com/aquasecurity/fanal/config/scanner.Scanner.ScanConfigs({{0x7ffe6831cf3b, 0x2b}, {0xc0004cd320, 0x2, 0x2}, 0xc0008746c0, 0xc00083c0e0, 0xc0009de230}, {0x21517d8, 0xc00010ff80}, ...)
        /home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220225095822-ef150f781751/config/scanner/scanner.go:90 +0x2d8
github.com/aquasecurity/fanal/artifact/local.Artifact.Inspect({{0x7ffe6831cf3b, 0x2b}, {0x7f29cee22a70, 0xc00010aa80}, {{{0x0, 0x0, 0x0}, {0xc000874700, 0x3, 0x4}}}, ...}, ...)
        /home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220225095822-ef150f781751/artifact/local/fs.go:115 +0x345
github.com/aquasecurity/trivy/pkg/scanner.Scanner.ScanArtifact({{_, _}, {_, _}}, {_, _}, {{0x0, 0x0, 0x0}, {0xc00010a820, ...}, ...})
        /home/runner/work/trivy/trivy/pkg/scanner/scan.go:94 +0xce
github.com/aquasecurity/trivy/pkg/commands/artifact.scan({_, _}, {{0xc0007e4680, 0xc000116a98, {0x20f7180, 0x6}, 0x1, 0x0, {0xc0002ee378, 0x12}}, ...}, ...)
        /home/runner/work/trivy/trivy/pkg/commands/artifact/run.go:217 +0x8fe
github.com/aquasecurity/trivy/pkg/commands/artifact.runWithTimeout({_, _}, {{0xc0007e4680, 0xc000116a98, {0x20f7180, 0x6}, 0x1, 0x0, {0xc0002ee378, 0x12}}, ...}, ...)
        /home/runner/work/trivy/trivy/pkg/commands/artifact/run.go:68 +0x44c
github.com/aquasecurity/trivy/pkg/commands/artifact.Run({_, _}, {{0xc0007e4680, 0xc000116a98, {0x20f7180, 0x6}, 0x1, 0x0, {0xc0002ee378, 0x12}}, ...}, ...)
        /home/runner/work/trivy/trivy/pkg/commands/artifact/run.go:40 +0xca
github.com/aquasecurity/trivy/pkg/commands/artifact.ConfigRun(0xc0007e4680)
        /home/runner/work/trivy/trivy/pkg/commands/artifact/config.go:29 +0x2af
github.com/urfave/cli/v2.(*Command).Run(0xc0000f7320, 0xc000779f40)
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/command.go:163 +0x64a
github.com/urfave/cli/v2.(*App).RunContext(0xc000325ba0, {0x21517a0, 0xc00012a020}, {0xc00011a040, 0x4, 0x4})
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/app.go:313 +0x81e
github.com/urfave/cli/v2.(*App).Run(...)
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/app.go:224
main.main()
        /home/runner/work/trivy/trivy/cmd/trivy/main.go:16 +0x4f
```
## What did you expect to happen?

Trivy would produce it's incredibly helpful security output message

## What happened instead?
It segfaulted. 

## Output of run with `-debug`:

```
2022-03-03T16:02:16.576Z        DEBUG   Severities: UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL
2022-03-03T16:02:16.580Z        DEBUG   cache dir:  /root/.cache/trivy
2022-03-03T16:02:16.580Z        DEBUG   Vulnerability type:  []
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xf33beb]

goroutine 1 [running]:
github.com/aquasecurity/defsec/rules.(*Result).Flatten(_)
        /home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.12.1/rules/flat.go:61 +0x32b
github.com/aquasecurity/fanal/config/scanner.Scanner.scanTerraform({{0x7ffdfbf28f3b, 0x2b}, {0xc0000caea0, 0x2, 0x2}, 0xc000d03d00, 0xc000723d50, 0xc0005debe0}, {0xc000a78000, 0xe, ...})
        /home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220221141823-58a5d668e45f/config/scanner/scanner.go:227 +0x425
github.com/aquasecurity/fanal/config/scanner.Scanner.ScanConfigs({{0x7ffdfbf28f3b, 0x2b}, {0xc0000caea0, 0x2, 0x2}, 0xc000d03d00, 0xc000723d50, 0xc0005debe0}, {0x21517d8, 0xc0008298c0}, ...)
        /home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220221141823-58a5d668e45f/config/scanner/scanner.go:90 +0x2d8
github.com/aquasecurity/fanal/artifact/local.Artifact.Inspect({{0x7ffdfbf28f3b, 0x2b}, {0x7f471330ed18, 0xc00068ca00}, {{{0x0, 0x0, 0x0}, {0xc000d03d40, 0x3, 0x4}}}, ...}, ...)
        /home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220221141823-58a5d668e45f/artifact/local/fs.go:115 +0x345
github.com/aquasecurity/trivy/pkg/scanner.Scanner.ScanArtifact({{_, _}, {_, _}}, {_, _}, {{0x0, 0x0, 0x0}, {0xc00068c920, ...}, ...})
        /home/runner/work/trivy/trivy/pkg/scanner/scan.go:94 +0xce
github.com/aquasecurity/trivy/pkg/commands/artifact.scan({_, _}, {{0xc000724b40, 0xc00000ea50, {0x20f7180, 0x6}, 0x0, 0x1, {0xc00079c870, 0x12}}, ...}, ...)
        /home/runner/work/trivy/trivy/pkg/commands/artifact/run.go:217 +0x8fe
github.com/aquasecurity/trivy/pkg/commands/artifact.runWithTimeout({_, _}, {{0xc000724b40, 0xc00000ea50, {0x20f7180, 0x6}, 0x0, 0x1, {0xc00079c870, 0x12}}, ...}, ...)
        /home/runner/work/trivy/trivy/pkg/commands/artifact/run.go:68 +0x44c
github.com/aquasecurity/trivy/pkg/commands/artifact.Run({_, _}, {{0xc000724b40, 0xc00000ea50, {0x20f7180, 0x6}, 0x0, 0x1, {0xc00079c870, 0x12}}, ...}, ...)
        /home/runner/work/trivy/trivy/pkg/commands/artifact/run.go:40 +0xca
github.com/aquasecurity/trivy/pkg/commands/artifact.ConfigRun(0xc000724b40)
        /home/runner/work/trivy/trivy/pkg/commands/artifact/config.go:29 +0x2af
github.com/urfave/cli/v2.(*Command).Run(0xc0001137a0, 0xc000724200)
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/command.go:163 +0x64a
github.com/urfave/cli/v2.(*App).RunContext(0xc000126340, {0x21517a0, 0xc000040060}, {0xc00003a080, 0x4, 0x4})
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/app.go:313 +0x81e
github.com/urfave/cli/v2.(*App).Run(...)
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/app.go:224
main.main()
        /home/runner/work/trivy/trivy/cmd/trivy/main.go:16 +0x4f
```

## Output of `trivy -v`:

```
0.24.0
```

## Additional details (base image name, container registry info...):
Docker Hub aquasec/trivy:0.24.0

Same issue using trivy 0.24.1, It works fine using the 0.23

```
trivy --debug conf .
2022-03-03T16:52:40.619-0300	DEBUG	Severities: UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL
2022-03-03T16:52:40.661-0300	DEBUG	cache dir:  /Users/krol/Library/Caches/trivy
2022-03-03T16:52:40.661-0300	DEBUG	Vulnerability type:  []
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x1b67f3a]

goroutine 1 [running]:
github.com/aquasecurity/defsec/providers/aws/autoscaling.(*LaunchConfiguration).RequiresIMDSToken(...)
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.12.1/providers/aws/autoscaling/autoscaling.go:30
github.com/aquasecurity/defsec/rules/aws/autoscaling.glob..func2(0xc0012b4000)
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.12.1/rules/aws/autoscaling/enforce_http_token_imds.go:45 +0x27a
github.com/aquasecurity/defsec/rules.RegisteredRule.Evaluate({{{0x28dd246, 0xc}, {0x2900764, 0x17}, {0x2973dd6, 0x4a}, {0x29a9881, 0x185}, {0x2963258, 0x37}, ...}, ...}, ...)
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.12.1/rules/register.go:24 +0x2a
github.com/aquasecurity/defsec/scanners/cloudformation/scanner.(*Scanner).Scan(0xc00069aaf0)
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.12.1/scanners/cloudformation/scanner/scanner.go:101 +0x3a5
github.com/aquasecurity/fanal/config/scanner.Scanner.scanCloudFormation({{0x7ff7bfeff89b, 0x1}, {0xc000d2a9e0, 0x2, 0x2}, 0xc000d130c0, 0xc000d10770, 0xc00069aaf0}, {0xc000e8cc00, 0x2, ...})
	/home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220225095822-ef150f781751/config/scanner/scanner.go:151 +0x297
github.com/aquasecurity/fanal/config/scanner.Scanner.ScanConfigs({{0x7ff7bfeff89b, 0x1}, {0xc000d2a9e0, 0x2, 0x2}, 0xc000d130c0, 0xc000d10770, 0xc00069aaf0}, {0x2d5a098, 0xc0006b5260}, ...)
	/home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220225095822-ef150f781751/config/scanner/scanner.go:97 +0x465
github.com/aquasecurity/fanal/artifact/local.Artifact.Inspect({{0x7ff7bfeff89b, 0x1}, {0xc85c338, 0xc000081810}, {{{0x0, 0x0, 0x0}, {0xc000d13100, 0x3, 0x4}}}, ...}, ...)
	/home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220225095822-ef150f781751/artifact/local/fs.go:115 +0x345
github.com/aquasecurity/trivy/pkg/scanner.Scanner.ScanArtifact({{_, _}, {_, _}}, {_, _}, {{0x0, 0x0, 0x0}, {0xc000080cd0, ...}, ...})
	/home/runner/work/trivy/trivy/pkg/scanner/scan.go:94 +0xce
github.com/aquasecurity/trivy/pkg/commands/artifact.scan({_, _}, {{0xc000bddcc0, 0xc000130410, {0x2cff304, 0x6}, 0x0, 0x1, {0xc000965700, 0x20}}, ...}, ...)
	/home/runner/work/trivy/trivy/pkg/commands/artifact/run.go:217 +0x8fe
github.com/aquasecurity/trivy/pkg/commands/artifact.runWithTimeout({_, _}, {{0xc000bddcc0, 0xc000130410, {0x2cff304, 0x6}, 0x0, 0x1, {0xc000965700, 0x20}}, ...}, ...)
	/home/runner/work/trivy/trivy/pkg/commands/artifact/run.go:68 +0x44c
github.com/aquasecurity/trivy/pkg/commands/artifact.Run({_, _}, {{0xc000bddcc0, 0xc000130410, {0x2cff304, 0x6}, 0x0, 0x1, {0xc000965700, 0x20}}, ...}, ...)
	/home/runner/work/trivy/trivy/pkg/commands/artifact/run.go:40 +0xca
github.com/aquasecurity/trivy/pkg/commands/artifact.ConfigRun(0xc000bddcc0)
	/home/runner/work/trivy/trivy/pkg/commands/artifact/config.go:29 +0x2af
github.com/urfave/cli/v2.(*Command).Run(0xc000d26360, 0xc000bdd580)
	/home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/command.go:163 +0x64a
github.com/urfave/cli/v2.(*App).RunContext(0xc000503d40, {0x2d5a060, 0xc0000440f0}, {0xc00003a080, 0x4, 0x4})
	/home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/app.go:313 +0x81e
github.com/urfave/cli/v2.(*App).Run(...)
	/home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/app.go:224
main.main()
	/home/runner/work/trivy/trivy/cmd/trivy/main.go:16 +0x4f
```

I am having the same issue scanning a CloudFormation template. Here is the error:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x28 pc=0x1058054e4]

goroutine 1 [running]:
github.com/aquasecurity/defsec/providers/aws/ec2.(*Instance).RequiresIMDSToken(...)
        /home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.12.1/providers/aws/ec2/instance.go:30
github.com/aquasecurity/defsec/rules/aws/ec2.glob..func2(0x1400053e800)
        /home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.12.1/rules/aws/ec2/enforce_http_token_imds.go:39 +0xd4
github.com/aquasecurity/defsec/rules.RegisteredRule.Evaluate({{{0x1061714ee, 0xc}, {0x106194fdb, 0x17}, {0x1062086d2, 0x4a}, {0x10623e029, 0x185}, {0x1061f7add, 0x37}, ...}, ...}, ...)
        /home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.12.1/rules/register.go:24 +0x34
github.com/aquasecurity/defsec/scanners/cloudformation/scanner.(*Scanner).Scan(0x140006a2b40)
        /home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.12.1/scanners/cloudformation/scanner/scanner.go:101 +0x384
github.com/aquasecurity/fanal/config/scanner.Scanner.scanCloudFormation({{0x16b0ab87b, 0x1f}, {0x140008fb600, 0x2, 0x2}, 0x14000776800, 0x1400096c540, 0x140006a2b40}, {0x1400055e0c0, 0x1, ...})
        /home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220303080309-254063f95ea0/config/scanner/scanner.go:151 +0x200
github.com/aquasecurity/fanal/config/scanner.Scanner.ScanConfigs({{0x16b0ab87b, 0x1f}, {0x140008fb600, 0x2, 0x2}, 0x14000776800, 0x1400096c540, 0x140006a2b40}, {0x1069e1af8, 0x140009688a0}, ...)
        /home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220303080309-254063f95ea0/config/scanner/scanner.go:97 +0x370
github.com/aquasecurity/fanal/artifact/local.Artifact.Inspect({{0x140000fa620, 0x1d}, {0x12f4b3a18, 0x140005989e0}, {{{0x0, 0x0, 0x0}, {0x14000776840, 0x3, 0x4}}}, ...}, ...)
        /home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220303080309-254063f95ea0/artifact/local/fs.go:115 +0x2c4
github.com/aquasecurity/trivy/pkg/scanner.Scanner.ScanArtifact({{0x1069b79e0, 0x140005e6f80}, {0x1069b7860, 0x140000fc000}}, {0x1069e1af8, 0x140009688a0}, {{0x0, 0x0, 0x0}, {0x14000598930, ...}, ...})
        /home/runner/work/trivy/trivy/pkg/scanner/scan.go:94 +0x74
github.com/aquasecurity/trivy/pkg/commands/artifact.scan({0x1069e1af8, 0x140009688a0}, {{0x140008f5c80, 0x14000122a50, {0x10659974c, 0x6}, 0x0, 0x0, {0x14000537410, 0x28}}, ...}, ...)
        /home/runner/work/trivy/trivy/pkg/commands/artifact/run.go:217 +0x6d8
github.com/aquasecurity/trivy/pkg/commands/artifact.runWithTimeout({0x1069e1af8, 0x140009688a0}, {{0x140008f5c80, 0x14000122a50, {0x10659974c, 0x6}, 0x0, 0x0, {0x14000537410, 0x28}}, ...}, ...)
        /home/runner/work/trivy/trivy/pkg/commands/artifact/run.go:68 +0x374
github.com/aquasecurity/trivy/pkg/commands/artifact.Run({0x1069e1ac0, 0x14000128008}, {{0x140008f5c80, 0x14000122a50, {0x10659974c, 0x6}, 0x0, 0x0, {0x14000537410, 0x28}}, ...}, ...)
        /home/runner/work/trivy/trivy/pkg/commands/artifact/run.go:40 +0xa0
github.com/aquasecurity/trivy/pkg/commands/artifact.ConfigRun(0x140008f5c80)
        /home/runner/work/trivy/trivy/pkg/commands/artifact/config.go:29 +0x264
github.com/urfave/cli/v2.(*Command).Run(0x140003efc20, 0x140008f5540)
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/command.go:163 +0x634
github.com/urfave/cli/v2.(*App).RunContext(0x14000503a00, {0x1069e1ac0, 0x14000128008}, {0x1400011c090, 0x3, 0x3})
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/app.go:313 +0x760
github.com/urfave/cli/v2.(*App).Run(...)
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/app.go:224
main.main()
        /home/runner/work/trivy/trivy/cmd/trivy/main.go:16 +0x80
```

Here is the template (which was generated by CDK):
```
{
  "Resources": {
    "VPCB9E5F0B4": {
      "Type": "AWS::EC2::VPC",
      "Properties": {
        "CidrBlock": "10.0.0.0/16",
        "EnableDnsHostnames": true,
        "EnableDnsSupport": true,
        "InstanceTenancy": "default",
        "Tags": [
          {
            "Key": "Name",
            "Value": "UbuntuGoofStack/VPC"
          }
        ]
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/Resource"
      }
    },
    "VPCPublicSubnet1SubnetB4246D30": {
      "Type": "AWS::EC2::Subnet",
      "Properties": {
        "CidrBlock": "10.0.0.0/19",
        "VpcId": {
          "Ref": "VPCB9E5F0B4"
        },
        "AvailabilityZone": "ap-southeast-2a",
        "MapPublicIpOnLaunch": true,
        "Tags": [
          {
            "Key": "aws-cdk:subnet-name",
            "Value": "Public"
          },
          {
            "Key": "aws-cdk:subnet-type",
            "Value": "Public"
          },
          {
            "Key": "Name",
            "Value": "UbuntuGoofStack/VPC/PublicSubnet1"
          }
        ]
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/PublicSubnet1/Subnet"
      }
    },
    "VPCPublicSubnet1RouteTableFEE4B781": {
      "Type": "AWS::EC2::RouteTable",
      "Properties": {
        "VpcId": {
          "Ref": "VPCB9E5F0B4"
        },
        "Tags": [
          {
            "Key": "Name",
            "Value": "UbuntuGoofStack/VPC/PublicSubnet1"
          }
        ]
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/PublicSubnet1/RouteTable"
      }
    },
    "VPCPublicSubnet1RouteTableAssociation0B0896DC": {
      "Type": "AWS::EC2::SubnetRouteTableAssociation",
      "Properties": {
        "RouteTableId": {
          "Ref": "VPCPublicSubnet1RouteTableFEE4B781"
        },
        "SubnetId": {
          "Ref": "VPCPublicSubnet1SubnetB4246D30"
        }
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/PublicSubnet1/RouteTableAssociation"
      }
    },
    "VPCPublicSubnet1DefaultRoute91CEF279": {
      "Type": "AWS::EC2::Route",
      "Properties": {
        "RouteTableId": {
          "Ref": "VPCPublicSubnet1RouteTableFEE4B781"
        },
        "DestinationCidrBlock": "0.0.0.0/0",
        "GatewayId": {
          "Ref": "VPCIGWB7E252D3"
        }
      },
      "DependsOn": [
        "VPCVPCGW99B986DC"
      ],
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/PublicSubnet1/DefaultRoute"
      }
    },
    "VPCPublicSubnet2Subnet74179F39": {
      "Type": "AWS::EC2::Subnet",
      "Properties": {
        "CidrBlock": "10.0.32.0/19",
        "VpcId": {
          "Ref": "VPCB9E5F0B4"
        },
        "AvailabilityZone": "ap-southeast-2b",
        "MapPublicIpOnLaunch": true,
        "Tags": [
          {
            "Key": "aws-cdk:subnet-name",
            "Value": "Public"
          },
          {
            "Key": "aws-cdk:subnet-type",
            "Value": "Public"
          },
          {
            "Key": "Name",
            "Value": "UbuntuGoofStack/VPC/PublicSubnet2"
          }
        ]
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/PublicSubnet2/Subnet"
      }
    },
    "VPCPublicSubnet2RouteTable6F1A15F1": {
      "Type": "AWS::EC2::RouteTable",
      "Properties": {
        "VpcId": {
          "Ref": "VPCB9E5F0B4"
        },
        "Tags": [
          {
            "Key": "Name",
            "Value": "UbuntuGoofStack/VPC/PublicSubnet2"
          }
        ]
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/PublicSubnet2/RouteTable"
      }
    },
    "VPCPublicSubnet2RouteTableAssociation5A808732": {
      "Type": "AWS::EC2::SubnetRouteTableAssociation",
      "Properties": {
        "RouteTableId": {
          "Ref": "VPCPublicSubnet2RouteTable6F1A15F1"
        },
        "SubnetId": {
          "Ref": "VPCPublicSubnet2Subnet74179F39"
        }
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/PublicSubnet2/RouteTableAssociation"
      }
    },
    "VPCPublicSubnet2DefaultRouteB7481BBA": {
      "Type": "AWS::EC2::Route",
      "Properties": {
        "RouteTableId": {
          "Ref": "VPCPublicSubnet2RouteTable6F1A15F1"
        },
        "DestinationCidrBlock": "0.0.0.0/0",
        "GatewayId": {
          "Ref": "VPCIGWB7E252D3"
        }
      },
      "DependsOn": [
        "VPCVPCGW99B986DC"
      ],
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/PublicSubnet2/DefaultRoute"
      }
    },
    "VPCPublicSubnet3Subnet631C5E25": {
      "Type": "AWS::EC2::Subnet",
      "Properties": {
        "CidrBlock": "10.0.64.0/19",
        "VpcId": {
          "Ref": "VPCB9E5F0B4"
        },
        "AvailabilityZone": "ap-southeast-2c",
        "MapPublicIpOnLaunch": true,
        "Tags": [
          {
            "Key": "aws-cdk:subnet-name",
            "Value": "Public"
          },
          {
            "Key": "aws-cdk:subnet-type",
            "Value": "Public"
          },
          {
            "Key": "Name",
            "Value": "UbuntuGoofStack/VPC/PublicSubnet3"
          }
        ]
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/PublicSubnet3/Subnet"
      }
    },
    "VPCPublicSubnet3RouteTable98AE0E14": {
      "Type": "AWS::EC2::RouteTable",
      "Properties": {
        "VpcId": {
          "Ref": "VPCB9E5F0B4"
        },
        "Tags": [
          {
            "Key": "Name",
            "Value": "UbuntuGoofStack/VPC/PublicSubnet3"
          }
        ]
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/PublicSubnet3/RouteTable"
      }
    },
    "VPCPublicSubnet3RouteTableAssociation427FE0C6": {
      "Type": "AWS::EC2::SubnetRouteTableAssociation",
      "Properties": {
        "RouteTableId": {
          "Ref": "VPCPublicSubnet3RouteTable98AE0E14"
        },
        "SubnetId": {
          "Ref": "VPCPublicSubnet3Subnet631C5E25"
        }
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/PublicSubnet3/RouteTableAssociation"
      }
    },
    "VPCPublicSubnet3DefaultRouteA0D29D46": {
      "Type": "AWS::EC2::Route",
      "Properties": {
        "RouteTableId": {
          "Ref": "VPCPublicSubnet3RouteTable98AE0E14"
        },
        "DestinationCidrBlock": "0.0.0.0/0",
        "GatewayId": {
          "Ref": "VPCIGWB7E252D3"
        }
      },
      "DependsOn": [
        "VPCVPCGW99B986DC"
      ],
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/PublicSubnet3/DefaultRoute"
      }
    },
    "VPCIsolatedSubnet1SubnetEBD00FC6": {
      "Type": "AWS::EC2::Subnet",
      "Properties": {
        "CidrBlock": "10.0.96.0/19",
        "VpcId": {
          "Ref": "VPCB9E5F0B4"
        },
        "AvailabilityZone": "ap-southeast-2a",
        "MapPublicIpOnLaunch": false,
        "Tags": [
          {
            "Key": "aws-cdk:subnet-name",
            "Value": "Isolated"
          },
          {
            "Key": "aws-cdk:subnet-type",
            "Value": "Isolated"
          },
          {
            "Key": "Name",
            "Value": "UbuntuGoofStack/VPC/IsolatedSubnet1"
          }
        ]
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/IsolatedSubnet1/Subnet"
      }
    },
    "VPCIsolatedSubnet1RouteTableEB156210": {
      "Type": "AWS::EC2::RouteTable",
      "Properties": {
        "VpcId": {
          "Ref": "VPCB9E5F0B4"
        },
        "Tags": [
          {
            "Key": "Name",
            "Value": "UbuntuGoofStack/VPC/IsolatedSubnet1"
          }
        ]
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/IsolatedSubnet1/RouteTable"
      }
    },
    "VPCIsolatedSubnet1RouteTableAssociationA2D18F7C": {
      "Type": "AWS::EC2::SubnetRouteTableAssociation",
      "Properties": {
        "RouteTableId": {
          "Ref": "VPCIsolatedSubnet1RouteTableEB156210"
        },
        "SubnetId": {
          "Ref": "VPCIsolatedSubnet1SubnetEBD00FC6"
        }
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/IsolatedSubnet1/RouteTableAssociation"
      }
    },
    "VPCIsolatedSubnet2Subnet4B1C8CAA": {
      "Type": "AWS::EC2::Subnet",
      "Properties": {
        "CidrBlock": "10.0.128.0/19",
        "VpcId": {
          "Ref": "VPCB9E5F0B4"
        },
        "AvailabilityZone": "ap-southeast-2b",
        "MapPublicIpOnLaunch": false,
        "Tags": [
          {
            "Key": "aws-cdk:subnet-name",
            "Value": "Isolated"
          },
          {
            "Key": "aws-cdk:subnet-type",
            "Value": "Isolated"
          },
          {
            "Key": "Name",
            "Value": "UbuntuGoofStack/VPC/IsolatedSubnet2"
          }
        ]
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/IsolatedSubnet2/Subnet"
      }
    },
    "VPCIsolatedSubnet2RouteTable9B4F78DC": {
      "Type": "AWS::EC2::RouteTable",
      "Properties": {
        "VpcId": {
          "Ref": "VPCB9E5F0B4"
        },
        "Tags": [
          {
            "Key": "Name",
            "Value": "UbuntuGoofStack/VPC/IsolatedSubnet2"
          }
        ]
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/IsolatedSubnet2/RouteTable"
      }
    },
    "VPCIsolatedSubnet2RouteTableAssociation7BF8E0EB": {
      "Type": "AWS::EC2::SubnetRouteTableAssociation",
      "Properties": {
        "RouteTableId": {
          "Ref": "VPCIsolatedSubnet2RouteTable9B4F78DC"
        },
        "SubnetId": {
          "Ref": "VPCIsolatedSubnet2Subnet4B1C8CAA"
        }
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/IsolatedSubnet2/RouteTableAssociation"
      }
    },
    "VPCIsolatedSubnet3Subnet96034237": {
      "Type": "AWS::EC2::Subnet",
      "Properties": {
        "CidrBlock": "10.0.160.0/19",
        "VpcId": {
          "Ref": "VPCB9E5F0B4"
        },
        "AvailabilityZone": "ap-southeast-2c",
        "MapPublicIpOnLaunch": false,
        "Tags": [
          {
            "Key": "aws-cdk:subnet-name",
            "Value": "Isolated"
          },
          {
            "Key": "aws-cdk:subnet-type",
            "Value": "Isolated"
          },
          {
            "Key": "Name",
            "Value": "UbuntuGoofStack/VPC/IsolatedSubnet3"
          }
        ]
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/IsolatedSubnet3/Subnet"
      }
    },
    "VPCIsolatedSubnet3RouteTableCB6A1FDA": {
      "Type": "AWS::EC2::RouteTable",
      "Properties": {
        "VpcId": {
          "Ref": "VPCB9E5F0B4"
        },
        "Tags": [
          {
            "Key": "Name",
            "Value": "UbuntuGoofStack/VPC/IsolatedSubnet3"
          }
        ]
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/IsolatedSubnet3/RouteTable"
      }
    },
    "VPCIsolatedSubnet3RouteTableAssociation754FC198": {
      "Type": "AWS::EC2::SubnetRouteTableAssociation",
      "Properties": {
        "RouteTableId": {
          "Ref": "VPCIsolatedSubnet3RouteTableCB6A1FDA"
        },
        "SubnetId": {
          "Ref": "VPCIsolatedSubnet3Subnet96034237"
        }
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/IsolatedSubnet3/RouteTableAssociation"
      }
    },
    "VPCIGWB7E252D3": {
      "Type": "AWS::EC2::InternetGateway",
      "Properties": {
        "Tags": [
          {
            "Key": "Name",
            "Value": "UbuntuGoofStack/VPC"
          }
        ]
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/IGW"
      }
    },
    "VPCVPCGW99B986DC": {
      "Type": "AWS::EC2::VPCGatewayAttachment",
      "Properties": {
        "VpcId": {
          "Ref": "VPCB9E5F0B4"
        },
        "InternetGatewayId": {
          "Ref": "VPCIGWB7E252D3"
        }
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/VPC/VPCGW"
      }
    },
    "SecurityGroupDD263621": {
      "Type": "AWS::EC2::SecurityGroup",
      "Properties": {
        "GroupDescription": "UbuntuGoofStack/SecurityGroup",
        "SecurityGroupEgress": [
          {
            "CidrIp": "0.0.0.0/0",
            "Description": "Allow all outbound traffic by default",
            "IpProtocol": "-1"
          }
        ],
        "VpcId": {
          "Ref": "VPCB9E5F0B4"
        }
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/SecurityGroup/Resource"
      }
    },
    "IAMRole24D9329E": {
      "Type": "AWS::IAM::Role",
      "Properties": {
        "AssumeRolePolicyDocument": {
          "Statement": [
            {
              "Action": "sts:AssumeRole",
              "Effect": "Allow",
              "Principal": {
                "Service": "ec2.amazonaws.com"
              }
            }
          ],
          "Version": "2012-10-17"
        },
        "ManagedPolicyArns": [
          {
            "Fn::Join": [
              "",
              [
                "arn:",
                {
                  "Ref": "AWS::Partition"
                },
                ":iam::aws:policy/AmazonSSMManagedInstanceCore"
              ]
            ]
          }
        ]
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/IAMRole/Resource"
      }
    },
    "InstanceInstanceProfile4EF8500F": {
      "Type": "AWS::IAM::InstanceProfile",
      "Properties": {
        "Roles": [
          {
            "Ref": "IAMRole24D9329E"
          }
        ]
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/ Instance/InstanceProfile"
      }
    },
    "InstanceC3112760": {
      "Type": "AWS::EC2::Instance",
      "Properties": {
        "AvailabilityZone": "ap-southeast-2a",
        "IamInstanceProfile": {
          "Ref": "InstanceInstanceProfile4EF8500F"
        },
        "ImageId": "ami-03f7e4dc0e8a0e8bf",
        "InstanceType": "t3.nano",
        "SecurityGroupIds": [
          {
            "Fn::GetAtt": [
              "SecurityGroupDD263621",
              "GroupId"
            ]
          }
        ],
        "SubnetId": {
          "Ref": "VPCPublicSubnet1SubnetB4246D30"
        },
        "Tags": [
          {
            "Key": "Name",
            "Value": "UbuntuGoofStack/ Instance"
          }
        ],
        "UserData": {
          "Fn::Base64": "#!/bin/bash"
        }
      },
      "DependsOn": [
        "IAMRole24D9329E"
      ],
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/ Instance/Resource"
      }
    },
    "CDKMetadata": {
      "Type": "AWS::CDK::Metadata",
      "Properties": {
        "Analytics": "v2:deflate64:H4sIAAAAAAAA/1WOQQ+CMAyFf4v3MRXjwaPxYLwtYLyaMUusQme2TkII/110IvH0Xr+2r03lci0XM934xFzuSYWF7HLW5i4GdO7ApLI7PYzYlXRSO6FCUaHJQ0HAbza5zAaGoy4qmPjEtt5bg5rR0m9YKIdPzTBlHIjBDX4/0Ea336PfasvDW9caiEUOJjjkdu9seHzO/YEDedZkIEZG3wvUtewyG/8bdWwrZ0usoO9FBt4GF5dH3wvV8tXSfCU3crmY3Txi4gIx1iCzqC8JpXZrSQEAAA=="
      },
      "Metadata": {
        "aws:cdk:path": "UbuntuGoofStack/CDKMetadata/Default"
      }
    }
  },
  "Parameters": {
    "BootstrapVersion": {
      "Type": "AWS::SSM::Parameter::Value<String>",
      "Default": "/cdk-bootstrap/hnb659fds/version",
      "Description": "Version of the CDK Bootstrap resources in this environment, automatically retrieved from SSM Parameter Store. [cdk:skip]"
    }
  },
  "Rules": {
    "CheckBootstrapVersion": {
      "Assertions": [
        {
          "Assert": {
            "Fn::Not": [
              {
                "Fn::Contains": [
                  [
                    "1",
                    "2",
                    "3",
                    "4",
                    "5"
                  ],
                  {
                    "Ref": "BootstrapVersion"
                  }
                ]
              }
            ]
          },
          "AssertDescription": "CDK bootstrap stack version 6 required. Please run 'cdk bootstrap' with a recent version of the CDK CLI."
        }
      ]
    }
  }
}
```

--- a/go.mod
+++ b/go.mod
@@ -7,7 +7,7 @@ require (
 	github.com/Masterminds/sprig v2.22.0+incompatible
 	github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46
 	github.com/aquasecurity/bolt-fixtures v0.0.0-20200903104109-d34e7f983986
-	github.com/aquasecurity/fanal v0.0.0-20220303131110-677831c145ce
+	github.com/aquasecurity/fanal v0.0.0-20220309115321-0183e4870f89
 	github.com/aquasecurity/go-dep-parser v0.0.0-20220302151315-ff6d77c26988
 	github.com/aquasecurity/go-gem-version v0.0.0-20201115065557-8eed6fe000ce
 	github.com/aquasecurity/go-npm-version v0.0.0-20201110091526-0b796d180798
@@ -32,7 +32,7 @@ require (
 	github.com/masahiro331/go-mvn-version v0.0.0-20210429150710-d3157d602a08
 	github.com/olekukonko/tablewriter v0.0.5
 	github.com/open-policy-agent/opa v0.37.2
-	github.com/owenrumney/go-sarif/v2 v2.0.17
+	github.com/owenrumney/go-sarif/v2 v2.1.1
 	github.com/package-url/packageurl-go v0.1.1-0.20220203205134-d70459300c8a
 	github.com/spf13/afero v1.8.1
 	github.com/stretchr/testify v1.7.0
@@ -73,8 +73,8 @@ require (
 	github.com/agext/levenshtein v1.2.3 // indirect
 	github.com/apparentlymart/go-cidr v1.1.0 // indirect
 	github.com/apparentlymart/go-textseg/v13 v13.0.0 // indirect
-	github.com/aquasecurity/defsec v0.14.0 // indirect
-	github.com/aquasecurity/tfsec v1.5.0 // indirect
+	github.com/aquasecurity/defsec v0.17.1 // indirect
+	github.com/aquasecurity/tfsec v1.8.0 // indirect
 	github.com/aws/aws-sdk-go v1.43.8 // indirect
 	github.com/bgentry/go-netrc v0.0.0-20140422174119-9fd32a8b3d3d // indirect
 	github.com/bmatcuk/doublestar v1.3.4 // indirect

--- a/integration/docker_engine_test.go
+++ b/integration/docker_engine_test.go
@@ -192,7 +192,7 @@ func TestDockerEngine(t *testing.T) {
 			name:         "sad path, invalid image",
 			invalidImage: true,
 			input:        "badimage:latest",
-			wantErr:      "unable to inspect the image (index.docker.io/library/badimage:latest)",
+			wantErr:      "unable to inspect the image (badimage:latest)",
 		},
 	}
 

